Automattic/wp-calypso
# Update/built by card for my home copy

#### Proposed Changes

Request link: pau2Xa-4nn-p2#comment-12884
**Headline copy change.** 

_Current headline copy:_
Hire experts to build your website

_New headline copy_
Get expert help for your website

#### Testing Instructions

- In calypso dev environment, switch to branch "update/built-by-card-for-my-home-copy"

- Make sure your user is set to English locale in {yoursite-test-site-domain}/me/account

- Navigate to the home page in calypso with the view parameter set to "VIEW_POST_LAUNCH" e.g
http://calypso.localhost:3000/home/{your-site-slug}?view=VIEW_POST_LAUNCH
**note**: replace {your-site-slug} with your site slug.

- Rotate through the marketing cards on the home page until you see the "built by" card.

- Verify that the card title is

"Get expert help for your website"

**Before**
<img width="1074" alt="card-b4" src="https://user-images.githubusercontent.com/11347472/202726827-2c3041e0-f5d8-4867-8cd5-e76bcab291f8.png">

**After**
<img width="1075" alt="card after" src="https://user-images.githubusercontent.com/11347472/202726909-2606f6ab-21e5-47eb-91b9-b5fc0a781fea.png">



*

#### Pre-merge Checklist





Related to #
